src/Joblify.Core.Api/Offers/OfferController.cs

```diff
@@ -60,6 +61,47 @@ public async Task<IActionResult> GetAllTrades()
             return Ok(response);
         }
 
+        [HttpGet("getOffersForUser")]
+        public async Task<IActionResult> GetOffersForUser(int id)
+        {
+            var response = await _offerService.GetOffersForUser(id);
+            if (response == null)
+            {
+                return BadRequest();
+            }
+            return Ok(response);
+        }
+
+        [HttpPut("UpdateOffer")]
+        public async Task<IActionResult> UpdateOffer(int id, [FromBody]EditOfferDto offer)
+        {
+            if (offer is null)
+            {
+                return BadRequest();
+            }
+
+            if (!ModelState.IsValid)
```

Comment:
I'm not sure if it's neccessary cause if you look at Infrastructure/ActionFilterAttributes/ we already have a class that checks ModelState for us

Reply:
Wow, that's nice. I didn't know that sth like this exist. thx for review